build.zig

```diff
@@ -1018,7 +1018,7 @@ const JniTestStep = struct {
                 else if (std.mem.indexOf(u8, ldd_result, "libc") != null)
                     .gnu
                 else {
-                    std.log.err("{s}", .{ ldd_result });
+                    std.log.err("{s}", .{ldd_result});
```

Comment:
@cb22 hm, haven't we fixed `zig fmt` not gating the main branch? This is pre-existing I think?

Reply:
Hmmm, it only runs on `src/`

Reply:
It looks like we only check the zig files in `src/`, so `build.zig` itself is not checked.